README.md

```diff
@@ -19,10 +19,11 @@ Start unit tests with `npm test`, `yarn run test`, or `docker-compose -f docker-
 Usage: dockerfilelint [files | content..] [options]
 
 Options:
-  -o, --output  Specify the format to use for output of linting results. Valid values
-                are `json` or `cli` (default).                               [string]
-  -j, --json    Output linting results as JSON, equivalent to `-o json`.    [boolean]
-  -h, --help    Show help                                                   [boolean]
+  -o, --output   Specify the format to use for output of linting results. Valid
+                 values are `json` or `cli` (default).                       [string]
+  -j, --json     Output linting results as JSON, equivalent to `-o json`.   [boolean]
+  -v, --version  Show version number                                        [boolean]
+  -h, --help     Show help                                                  [boolean]
```

Comment:
Thanks for updating this too.

How would you feel about using `.wrap(86)` on line 28 of bin/dockerfilelint instead? That would slightly expand the output to maintain the illusion of right-alignment, which looks like this:

```
Options:
  -o, --output   Specify the format to use for output of linting results. Valid values
                 are `json` or `cli` (default).                               [string]
  -j, --json     Output linting results as JSON, equivalent to `-o json`.    [boolean]
  -v, --version  show version information                                    [boolean]
  -h, --help     Show help                                                   [boolean]
```